.github/actions/setup-cli/action.yaml

```diff
@@ -0,0 +1,35 @@
+# action.yml
+name: 'Setup Tools'
```

Comment:
there even greater common denominator - checking out code, setting up go, etc... we should extract all this in this common action.